app/components/FormikCodeInput/index.js

```diff
@@ -113,6 +113,7 @@ function FormikCodeInput({ formikKey, label, codeLength }) {
                 width="95%"
                 maxLength="1"
                 size="1"
+                keyboard="tel"
```

Comment:
Are you sure this change will not break any UI like displaying browser-native arrows in the inputs? Did you test it? I would set `inputmode` instead if this change breakes anything. (See that `keyboard` props sets input's `type` attribute)